src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -135,6 +135,16 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
         assertTrue(hasStudent1ReceivedEmail);
         assertTrue(hasInstructor1ReceivedEmail);
         
+        ______TS("feedback session closed alerts");
+        
+        emails = new EmailGenerator().generateFeedbackSessionClosedEmails(session);
+        assertEquals(5, emails.size());
+        
+        subject = String.format(EmailType.FEEDBACK_CLOSED.getSubject(),
+                                course.getName(), session.getFeedbackSessionName());
+        
+        verifyEmail(emails.get(1), instructor1.email, subject, "/sessionClosedEmailForInstructor.html");
```

Comment:
This is not a good way to test as it's subject to an unpredictable ordering. Instead, look at how the other tests do it in order to avoid this randomness.